__tests__/api.test.js

```diff
@@ -86,3 +85,65 @@ describe("4. GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("5. PATCH /api/articles/:article_id", () => {
+  test("should return status 200 ", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200);
+  });
+  test("should return updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 50 })
+      .then(({ body }) => {
+        const expectedArticle = {
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 150,
+        };
+        expect(body.article[0]).toEqual(expectedArticle);
+      });
+  });
+  test("should respond with 400 invalid id when given invalid author_id", () => {
+    return request(app)
+      .patch("/api/articles/help")
+      .send({ inc_votes: 50 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid id");
+      });
+  });
+  test("should respond with 406 not acceptable when sent malformed body - too many fields ", () => {
```

Comment:
I'd say this is a 400. 406 is an error associated with the _accept header_.

Reply:
also interface we're making is PATCH /api/articles/1

if we wanted to add the functionality of updating the body of the article, it would be on this same endpoint.
This test would then likely fail.